src/components/ProductItem/ProductPrice.tsx

```diff
@@ -137,84 +140,86 @@ export const ProductPrice: FunctionComponent<ProductPriceProps> = ({
-                <span className="text-brand-600">
-                  {getProductPrice(
-                    item,
-                    currencySymbol,
-                    currencyRate,
-                    false,
-                    true
-                  )}
-                </span>
-              </p>
-            )}
+  const renderPriceWithLabel = (price: string | ReactNode) => {
+    const priceClass = priceLabel ? 'line-through' : '';
 
-          {!isBundle &&
-            !isGrouped &&
-            !isGiftCard &&
-            !isConfigurable &&
-            !isComplexProductView &&
-            !discount && (
-              <p className="ds-sdk-product-price--no-discount font-headline-4-strong">
-                {getProductPrice(
-                  item,
-                  currencySymbol,
-                  currencyRate,
-                  false,
-                  true
-                )}
-              </p>
+    return (
+      <span className="price-wrapper">
+        <span className={priceClass}>{price}</span>
+        {priceLabel && (
+          <span className="ml-2 text-neutral-700">{priceLabel.txt}</span>
+        )}
+      </span>
+    );
+  };
+
+  const priceContent = (() => {
+    if (!price) return null;
+
+    if (isBundle) {
+      return (
+        <div className="ds-sdk-product-price--bundle">
+          <p className="mt-xs font-headline-4-default">
+            {renderPriceWithLabel(
+              getBundledPrice(item, currencySymbol, currencyRate)
             )}
+          </p>
+        </div>
+      );
+    }
 
-          {isBundle && (
-            <div className="ds-sdk-product-price--bundle">
-              <p className="mt-xs font-headline-4-default">
-                {getBundledPrice(item, currencySymbol, currencyRate)}
-              </p>
-            </div>
+    if (isGrouped) {
+      return (
+        <p className="ds-sdk-product-price--grouped font-headline-4-strong">
+          {renderPriceWithLabel(
+            getPriceFormat(item, currencySymbol, currencyRate, false)
           )}
+        </p>
+      );
+    }
 
-          {isGrouped && (
-            <p className="ds-sdk-product-price--grouped font-headline-4-strong">
-              {getPriceFormat(item, currencySymbol, currencyRate, false)}
-            </p>
+    if (isGiftCard) {
+      return (
+        <p className="ds-sdk-product-price--gift-card font-headline-4-strong">
+          {renderPriceWithLabel(
+            getPriceFormat(item, currencySymbol, currencyRate, true)
           )}
+        </p>
+      );
+    }
 
-          {isGiftCard && (
-            <p className="ds-sdk-product-price--gift-card font-headline-4-strong">
-              {getPriceFormat(item, currencySymbol, currencyRate, true)}
-            </p>
-          )}
+    if (isConfigurable || isComplexProductView) {
+      return (
+        <p className="ds-sdk-product-price--configurable font-headline-4-strong">
+          {renderPriceWithLabel(getDiscountedPrice(discount))}
```

Comment:
Do we need strikethrough for whole price ?
e.g. the price is displayed as
1. regular price as strikethrough and final price for discount product
2. Starting at $10   $20 for bundle product

Can you please check with Erika how it should look ?
You can follow this thread for the previous conversation 
https://adobe-dx-support.slack.com/archives/C04CQH83BME/p1719990986003769